README.md

```diff
@@ -93,19 +93,33 @@ $ COSMOS_NODE=localhost yarn testnet
 ---
 
 ### Production
-Get the Gaia binary from [GitHub](`https://github.com/cosmos/gaia/releases`).
 
-Install [Docker](https://docs.docker.com/get-started/).
+Building requires that [Docker](https://www.docker.com/get-docker) is installed
+on your system.
 
-Build and run the app.
+Build the app:
+
+The first argument is a commit for Voyager (e.g., the tag `0.4.3`).  Optional
+arguments (such as `--platform`) must come afterwards.
+
+```bash
+yarn run build 0.4.3 --platform={darwin|win32|linux}
+```
+
+You can specify a commit hash for the version of Gaia to use if it has been prebuilt:
+
+```bash
+GAIA_COMMIT=decb23bed0179244e4f42d697dcd4bf759200a2d yarn run build 0.4.3
+```
+
+Run the app.
 ```bash
-yarn run build --platform={darwin|win32|linux} -- --binary={path to the gaia binary}
 open builds/Cosmos-{platform}-x64/Cosmos.app
 ```
 
 When you are testing the build system you can skip the repackaging of the JS files.
 ```bash
-$ yarn run build --platform= --{darwin|win32|linux} --skip-pack --binary=...
+$ yarn run build 0.4.3 --platform=darwin --skip-pack
```

Comment:
Is the here specified version used somewhere? What do you think about taking it from the package.json?

Reply:
It's not a version, it's a commit reference.  It's explained a few lines earlier in the README:

> The first argument is a commit for Voyager (e.g., the tag `0.4.3`).  Optional
arguments (such as `--platform`) must come afterwards.

I added this in response to your request:

> Maybe we can make the script take a git-tag for Voyager and then it pulls the code according to the tag.

Reply:
Thanks for clarifying this.